containers/Investment/RevolvingPool/View/InvestCard.tsx

```diff
@@ -56,25 +72,39 @@ const InvestCard: React.FC<Props> = (props: Props) => {
 
   const disabled = status === 'unconfirmed' || status === 'pending'
 
+  const [error, setError] = React.useState<string | undefined>(undefined)
+
+  const onChange = (newValue: string) => {
+    setDaiValue(newValue)
+    if (hasInvested === false && new BN(newValue).lt(MinInvestment)) {
+      setError(`Minimum investment: ${config.network === 'Mainnet' ? '10.000' : '10'} DAI`)
+    } else if (limit && new BN(newValue).gt(new BN(limit))) {
+      setError('Amount larger than balance')
+    } else if (new BN(newValue).isZero()) {
+      setError('')
```

Comment:
should this have an error statement?

Reply:
Yeah that's a bit of hack because I want to get the red line in the input (as in, get it into an error state), without actually showing an error message. I'm not sure I can think of an easy, better way.